src/main/java/com/hydratereminder/command/hydrate/HydrateCommandHandler.java

```diff
@@ -1,15 +1,32 @@
 package com.hydratereminder.command.hydrate;
 
-import javax.inject.Singleton;
-
+import com.hydratereminder.HydrateReminderPlugin;
+import com.hydratereminder.chat.ChatMessageSender;
 import com.hydratereminder.command.CommandHandler;
 
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
 @Singleton
 public class HydrateCommandHandler implements CommandHandler {
 
+    @Inject
+    ChatMessageSender chatMessageSender;
```

Comment:
You can use constructor based injection and make those variables `private final`